build/yaml/crd/vpc/crd.nsx.vmware.com_subnetports.yaml

```diff
@@ -59,6 +59,10 @@ spec:
                 description: SubnetSet defines the parent SubnetSet name of the SubnetPort.
                 type: string
             type: object
+            x-kubernetes-validations:
+            - message: Only one of subnet or subnetSet can be specified
+              rule: has(self.subnet) && !has(self.subnetSet) || !has(self.subnet)
```

Comment:
This rule does not include both subnet and subnetset empty case

Reply:
if both subnet and subnetset empty,  has(self.subnet) && !has(self.subnetSet) || !has(self.subnet) && has(self.subnetSet) cannot be true.